framework/src/outputs/ConsoleUtils.C

```diff
@@ -81,22 +81,64 @@ outputMeshInformation(FEProblemBase & problem, bool verbose)
     // clang-format on
   }
 
-  oss << std::setw(console_field_width) << "  Nodes:" << '\n'
-      << std::setw(console_field_width) << "    Total:" << mesh.n_nodes() << '\n'
-      << std::setw(console_field_width) << "    Local:" << mesh.n_local_nodes() << '\n'
-      << std::setw(console_field_width) << "  Elems:" << '\n'
-      << std::setw(console_field_width) << "    Total:" << mesh.n_active_elem() << '\n'
-      << std::setw(console_field_width) << "    Local:" << mesh.n_active_local_elem() << '\n';
+  dof_id_type min_nnodes = std::numeric_limits<dof_id_type>::max(), max_nnodes = 0;
+  dof_id_type min_nelems = std::numeric_limits<dof_id_type>::max(), max_nelems = 0;
+  if (mesh.n_processors() > 1)
+  {
+    std::vector<dof_id_type> n_local_nodes;
+    mesh.comm().gather(0, mesh.n_local_nodes(), n_local_nodes);
```

Comment:
I would just use  `mesh.comm().max()` and `mesh.comm().min()`

Reply:
lol. You are right. I was evaluating RMS using this before. But if we just need max and min, we do not need this.